ansible/roles/locate_env_authorized_key/tasks/main.yaml

```diff
@@ -30,11 +38,17 @@
     creates: "{{ hostvars.localhost.env_authorized_key_path_pub }}"
   delegate_to: localhost
   become: false
+  when:
+    - not env_authorized_key_pub_stats.stat.exists
```

Comment:
Isn't the `args.creates` parameter enough here?

Reply:
Yes, `args.creates` is enough, fixed with 7aa99f3950649176357bc49008992c3de431fcee

Reply:
Documentation updated with 394dadf3393b4ae03eefc8d27f4e5c8adc8b628e